src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -89,13 +89,15 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
 
     lv_layer_t * layer = u->base_unit.target_layer;
 
-    lv_vg_lite_buffer_init(
-        &u->target_buffer,
-        layer->buf,
-        lv_area_get_width(&layer->buf_area),
-        lv_area_get_height(&layer->buf_area),
-        lv_vg_lite_vg_fmt(layer->color_format),
-        false);
+    lv_draw_buf_t draw_buf = { 0 };
+    uint32_t w, h, stride;
+    w = lv_area_get_width(&layer->buf_area);
+    h = lv_area_get_height(&layer->buf_area);
+    stride = lv_draw_buf_width_to_stride(w, layer->color_format);
+
+    lv_image_header_init(&draw_buf.header, w, h, layer->color_format, stride, 0);
+    draw_buf.data = layer->buf;
+    lv_vg_lite_buffer_from_draw_buf(&u->target_buffer, &draw_buf);
```

Comment:
In future, when layer will migrate to draw_buf too, this will be simplified as below.
```suggestion
    lv_vg_lite_buffer_from_draw_buf(&u->target_buffer, layer->draw_buf);
```

Reply:
Yes, there is no rush to use the draw_buf structure here, which can be fixed along with the lv_layer changes.